config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -85,7 +85,7 @@
 # serializer. Therefore, this setting should only be enabled after all replicas
 # have been successfully upgraded to Rails 7.1.
 #++
-# Rails.application.config.active_job.use_big_decimal_serializer = true
+Rails.application.config.active_job.use_big_decimal_serializer = true
```

Comment:
We do appear to have jobs in `app/jobs` so I wonder whether this requires more investigation to be confident that it's continuing to work as expected?

Reply:
You're right @chrisroos - there is a single job. I've updated the commit message to read: 

> Accept the default for this option. We only have a single job
UploadJob which takes a hash argument so I don't think we need to
worry about the comment here which suggests we perform a rolling
upgrade to avoid issues with serialized BigDecimal arguments.